pkgs/development/python-modules/robotstatuschecker/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, python, robotframework }:
+
+buildPythonPackage rec {
+  version = "1.3";
+  pname = "robotstatuschecker";
+
+  # no tests included in PyPI tarball
+  src = fetchFromGitHub {
+    owner = "robotframework";
+    repo = "statuschecker";
+    rev = version;
+    sha256 = "0rppwwpp4djn5c43x7icwslnxbzcfnnn3c6awpg1k97j69d2nmln";
+  };
+
+  propagatedBuildInputs = [ robotframework ];
+
+  checkPhase = ''
+    ${python.interpreter} test/run.py
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A tool for checking that Robot Framework test cases have expected statuses and log messages";
+    homepage = https://github.com/robotframework/statuschecker;
+    license = licenses.mit;
```

Comment:
Isn't it Apache 2.0-licensed?

See [comment in `robotstatuschecker.py`](https://github.com/robotframework/statuschecker/blob/1.3/robotstatuschecker.py#L5), [`License` classifier in `setup.py`](https://github.com/robotframework/statuschecker/blob/1.3/setup.py#L14) and [PyPI](https://pypi.org/project/robotstatuschecker/).

Reply:
you are right! fixed thanks!